exonum-java-binding/service-archetype/src/main/resources/archetype-resources/start-testnet.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# Starts a test Exonum network with a single validator node.
+#
+# The network files are placed in `$PWD/testnet` directory, which is cleared at the beginning.
+# The Java service artifacts must be placed into 'testnet/artifacts' directory.
+#
+# See https://exonum.com/doc/version/latest/get-started/java-binding/#node-configuration
```

Comment:
nit: why `latest` version of the docs is used here?

Reply:
The version in the generated script is not likely to be updated on Exonum version update in the POM, so both latest and a particular version could be incorrect.